react/src/components/VFolderNodes.tsx

```diff
@@ -237,15 +251,20 @@ const VFolderNodes: React.FC<VFolderNodesProps> = ({
                         icon={<RestoreIcon />}
                         style={{
                           color:
-                            vfolder?.status !== 'delete-pending'
+                            vfolder?.status !== 'delete-pending' ||
+                            usageMode === 'data'
```

Comment:
Consider assigning this expression to a variable to improve readability and reduce repetition. This also helps prevent potential logic inconsistencies or leaks if the expression changes in one place but not the other.
```suggestion
                            isPipelineFolder // usageMode === 'data'
```